include.js

```diff
@@ -13,34 +13,41 @@ let cards = [
     author: "ArthurDoom",
     githubLink: "https://github.com/ArthurDoom"
   },
-	{
```

Comment:
Here some of the changes are due to prettier which automatically formatted the file correctly by removing some unneeded space. Thank you.